src/Api/Controller/ListUsersController.php

```diff
@@ -78,6 +79,8 @@ protected function data(ServerRequestInterface $request, Document $document)
         $offset = $this->extractOffset($request);
         $load = $this->extractInclude($request);
 
+        $document->addMeta('totalCount', User::count());
```

Comment:
I see you've come running back! :smile:

Reply:
Shhh! I was trying to hide it, then write a miraculous victory speech! You've exposed me now!

Reply:
According to all known laws
of computer science,

  
there is no way a DavWheat
should be able to PHP.

  
Its Javascript focus is too intense to get
its logic to the backend.

  
The DavWheat, of course, PHPs anyway

  
because DavWheats don't care
what computer scientists think is impossible.



... 
I'll see myself out.

Reply:
That was scarily accurate! 😂

Are you *sure* you don't want to become a ~~bee movie plagiarist~~ writer in the future? :P